src/components/navigation.jsx

```diff
@@ -1,18 +1,25 @@
 import React from 'react';
+import { FaUser } from 'react-icons/fa';
 import { Link } from 'react-router-dom';
 
 const Navigation = () => (
-  <nav className="headernav">
-    <h2 className="logo">Bookstore CMS</h2>
-    <ul>
-      <li>
-        <Link className="nav books" to="/">Books</Link>
-      </li>
-      <li>
-        <Link className="nav category" to="/authors">Authors</Link>
-      </li>
-    </ul>
-  </nav>
+  <header>
+    <nav>
+      <div className="nav-container">
+        <h2 className="logo">Bookstore CMS</h2>
+        <div>
+          <Link className="nav books" to="/">Books</Link>
+        </div>
+        <div>
+          <Link className="nav category" to="/category">Category</Link>
+        </div>
```

Comment:
- [ ] Good job so far 👍🏽 . I think you should kindly increase the font size of your navigations and capitalize the font to match the  Zeplin design

Your design 👇🏽 
![image](https://github.com/mershark/bookstore/assets/63473584/1074e3b8-f941-4727-8b64-49e6c434a273)

Zeplin design 👇🏽 
![image](https://github.com/mershark/bookstore/assets/63473584/0e3ea196-12f1-4914-886c-67d3d69fecbb)